mern/client/src/screens/StartingIntent/StartingIntent.js

```diff
@@ -10,22 +11,36 @@ function StartingIntent() {
   const PageChange = () => {
     Navigate("/");
   };
+
+  // dummy intent for testing purposes
+  const intent = [
+    {
+      value: "cheese",
+      included: false,
+      children: ["00000001", "00000010"],
+    },
+    {
+      value: "pepperoni",
+      included: false,
+      children: ["00000011", "00000100"],
+    },
+    {
+      value: "hawaiian",
+      included: false,
+      children: ["00000101", "00000110"],
+    },
+  ];
+
   return (
     <div className="container">
-      <div className={styles.scroller}>
```

Comment:
I think this is should be left as styles.scroller because we made changes to the css files in our recent PRs. 
You probably have to pull from integration for this!!

Reply:
Waittt I realized you did pull the new changes from integration. Can we just call this without the styles.?